capstone/src/main/java/com/google/sps/AssociationResult.java

```diff
@@ -42,6 +47,7 @@ public AssociationResult(String content, float score, float weight) {
     this.content = content;
     this.score = score;
     this.weight = weight;
+    strongSentiment = Math.abs(score) > SENTIMENT_THRESHOLD;
```

Comment:
Should we do `this.strongSentiment` to be consistent with the other class members?

Reply:
Fixed!